service/main.go

```diff
@@ -134,56 +154,64 @@ func (s *Service) NewEnvelope(mtype pb.Message_Type, msg proto.Message, id *int3
 		}
 	}
 	message := &pb.Message{Type: mtype, Payload: payload}
+
 	if id != nil {
 		message.RequestId = *id
 	}
 	if response {
 		message.IsResponse = true
 	}
+
 	ser, err := proto.Marshal(message)
 	if err != nil {
 		return nil, err
 	}
-	if s.Node.PrivateKey == nil {
-		if err := s.Node.LoadPrivateKey(); err != nil {
+
+	if srv.Node.PrivateKey == nil {
+		if err := srv.Node.LoadPrivateKey(); err != nil {
 			return nil, err
 		}
 	}
-	sig, err := s.Node.PrivateKey.Sign(ser)
+
+	sig, err := srv.Node.PrivateKey.Sign(ser)
 	if err != nil {
 		return nil, err
 	}
+
 	return &pb.Envelope{Message: message, Sig: sig}, nil
 }
 
 // NewError returns a signed pb error message
-func (s *Service) NewError(code int, msg string, id int32) (*pb.Envelope, error) {
-	return s.NewEnvelope(pb.Message_ERROR, &pb.Error{
+func (srv *Service) NewError(code int, msg string, id int32) (*pb.Envelope, error) {
+	return srv.NewEnvelope(pb.Message_ERROR, &pb.Error{
 		Code:    uint32(code),
 		Message: msg,
 	}, &id, true)
 }
 
 // VerifyEnvelope verifies the authenticity of an envelope
-func (s *Service) VerifyEnvelope(env *pb.Envelope, pid peer.ID) error {
+func (srv *Service) VerifyEnvelope(env *pb.Envelope, pid peer.ID) error {
 	ser, err := proto.Marshal(env.Message)
 	if err != nil {
 		return err
 	}
+
 	pk, err := pid.ExtractPublicKey()
 	if err != nil {
 		return err
 	}
+
 	return crypto.Verify(pk, ser, env.Sig)
 }
 
 // handleError receives an error response
-func (s *Service) handleError(env *pb.Envelope) error {
+func (srv *Service) handleError(env *pb.Envelope) error {
 	if env.Message.Payload == nil && env.Message.Type != pb.Message_PONG {
 		err := fmt.Sprintf("message payload with type %s is nil", env.Message.Type.String())
 		log.Error(err)
 		return errors.New(err)
 	}
+
```

Comment:
hah, most of these changes are just formatting and variable name changes! nice :P